### PR TITLE
Adding virtual sign-in option to the instructions

### PR DESCRIPTION
**Pull request by dylansdaniels**
@jasmainak I think this will be easier than a physical sign in sheet... plus I didn't have time to print one out 😅

### CODE REVIEW

**Comment by dylansdaniels** [comment]
instructions for downloading hnn-workshop-materials are currently pointing to Mainak's fork but I can change the link once that PR is merged if we want, or we can leave it as is

File: workshop-instructions/instructions.md [hunk 1]
```diff
@@ -14,28 +15,37 @@
...
+
+Click [here](https://github.com/jonescompneurolab/hnn-data/archive/refs/heads/main.zip) to download the hnn-data folder, and [here](https://github.com/jasmainak/hnn-workshop-materials/archive/refs/heads/biomag_2024.zip) to download the hnn-workshop-materials folder. 
+
+Altenatively, you can clone the repositores with the follwing commands.
 
 ```bash
-cd hnn-data
-git pull
+git clone https://github.com/jonescompneurolab/hnn-data.git
+git clone https://github.com/jasmainak/hnn-workshop-materials.git
```
Comment (jasmainak) [inline workshop-instructions/instructions.md]: I am working mostly on the colab as of now ... in two minds whether to abondon this altogether. But colab is really slow ...

Comment (jasmainak) [reply workshop-instructions/instructions.md]: let's just drop this github url for now ... focus on colab. I'll add the github URL at the start of the colab if it turns out to be something users can download

**Comment by dylansdaniels** [comment]
@jasmainak reworked the sections a bit. How does this look?

**Comment by jasmainak** [comment]
Looks good! Feel free to merge :)

**Pull request merged**